db/migrate/20160630133409_add_slug_to_nodes.rb

```diff
@@ -0,0 +1,11 @@
+class AddSlugToNodes < ActiveRecord::Migration
```

Comment:
Here can you update the timestamp of the migration so that it falls in the correct order? Just 201608009... would be perfect.